docs/app/blocks/components/live-example-block/live-example-block.component.scss

```diff
@@ -1,4 +1,76 @@
+// colors
+$color-white-default: #ffffff;
```

Comment:
let's leave for now, but we would need to move this to Nebular themes at some point